packages/core/src/messages/cache-info.ts

```diff
@@ -1,11 +1,94 @@
+type CacheLimitsProps = {
+  maxTrafficRate: number;
+  maxThroughputKbps: number;
+  maxItemSizeKb: number;
+  maxTtlSeconds: number;
+};
+
+export class CacheLimits {
```

Comment:
nit: I'm guessing you did these as classes because the main CacheInfo object is a cache; but is there any reason to do these as classes instead of interfaces?

Reply:
yea I think an interface should work here as well

Reply:
I feel like that might be a little simpler/cleaner, wdyt?

I know it's not a huge deal either way but since this is technically going to be public API, just wanted to make sure our decision was deliberate.